### PR TITLE
Logserver docker-compose project

### PR DESCRIPTION
**Pull request by ivan-c**
- Add logserver repo as git submodule
- Add docker-compose overrides for traefik ingress (`logs.$BASE_DOMAIN`)

### CODE REVIEW

File: dev/freestanding/logs/deploy-logs.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+if [ "$1" = -h ] || [ "$1" = --help ]; then
+    usage
+    exit 0
+fi
+
+
+# docker-compose commands must be run in the same directory as docker-compose.yaml
+cd "${repo_path}"
+
+echo "📦 Updating images..."
```
Comment (pbugni) [inline dev/freestanding/logs/deploy-logs.sh:27]: fancy :)

File: dev/freestanding/logs/deploy-logs.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+docker-compose pull
+
+echo "🚀 Deploying containers..."
```
Comment (pbugni) [inline dev/freestanding/logs/deploy-logs.sh:30]: yay, rocket emojis at last!!!

**Review by pbugni — APPROVED** [review]
Awesome, thank you.

File: dev/freestanding/logs/deploy-logs.sh (hunk 2 above)
Comment (ivan-c) [reply dev/freestanding/logs/deploy-logs.sh:30]: They actually serve a purpose! It's easy to differentiate output from the script vs output from the commands the script calls if I use emojis for the former

**Review by pbugni — APPROVED** [review]
Works great.  Thank you.

**Pull request merged**